hw11-cache/src/main/java/org/otus/education/cachehw/MyCache.java

```diff
@@ -0,0 +1,64 @@
+package org.otus.education.cachehw;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.WeakHashMap;
+
+/**
+ * @author sergey
+ * created on 14.12.18.
+ */
+public class MyCache<K, V> implements HwCache<K, V> {
+
+    private final WeakHashMap<K,V> hashMap;
```

Comment:
Называть переменную по имени типа данных плохо.
Надо называть не по структуре данных, которая там хранится (она может измениться), а по назначению переменной.
И объявлять лучше интерфейсами.


```suggestion
    private final Map<K,V> storage = new WeakHashMap<>();
```